packages/perseus/src/widgets/matrix/validate-matrix.ts

```diff
@@ -0,0 +1,30 @@
+import type {PerseusScore} from "../../types";
+import type {
+    PerseusMatrixUserInput,
+    PerseusMatrixValidationData,
+} from "../../validation.types";
+
+/**
+ * Checks user input from the matrix widget to see if it is scorable.
+ *
```

Comment:
Question: Would it be good to leave comments like this for the other validators or is this specifically because of the Note below this?

Reply:
🤔  I think it's worth a short description.